recipe/meta.yaml

```diff
@@ -17,6 +17,8 @@ build:
 
 requirements:
   run_constrained:
+    # For cpython>=3.13, we will make python package depend on this package.
+    # Therefore there is no need to distinguish freethreading or not in the build string
```

Comment:
@jjhelmus, I was thinking of making `python` depend on `python_abi`. Then the freethreading version will depend on `cp313t` version and the default build will depend on `cp313`. Then we don't have to duplicate the info in the build_string in `python` and everything is in `python_abi`.

Reply:
Having `python` depend on `python_abi` make sense and I can talk to people at Anaconda about adding that to our Python 3.13 release.

The free-threading build of `python` will likely need to be indicated in some manner to allow the variant to be selected in `conda_build_config.yaml` when building downstream packages.

Reply:
One option would be to add the ABI tag or another marker before `cpython` in the build string for the python package similar to PyPy. Then the build matrix would be something like:
```
python:
... 
  - 3.13.* *_cpython
  - 3.13.* *_cp313t_cpython
 ```

Reply:
Yeah, let's go with that.